custom_components/tahoma/__init__.py

```diff
@@ -90,21 +89,50 @@ async def async_setup(hass: HomeAssistant, config: dict):
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry):
```

Comment:
Works like a charm! Doesn't make this PR breaking anymore, and easy to remove for core PR.

```
2021-08-09 22:41:33 DEBUG (MainThread) [custom_components.tahoma] Migrating from version 1
2021-08-09 22:41:33 INFO (MainThread) [custom_components.tahoma] Migrated Somfy TaHoma to somfy_europe
2021-08-09 22:41:33 INFO (MainThread) [custom_components.tahoma] Migration to version 2 successful
```

Reply:
Nice!